attestation-service/attestation-service/src/token/simple.rs

```diff
@@ -39,10 +47,12 @@ impl SimpleAttestationTokenBroker {
 
 impl SimpleAttestationTokenBroker {
     fn rs384_sign(&self, payload: &[u8]) -> Result<Vec<u8>> {
-        let mut rng = rand::thread_rng();
-        let signing_key = SigningKey::<Sha384>::new(self.private_key.clone());
-        let signature = signing_key.sign_with_rng(&mut rng, payload);
-        Ok(signature.to_bytes().to_vec())
+        let rsa_pkey = PKey::from_rsa(self.private_key.clone())?;
```

Comment:
This is because when AS generates the JWK in Token, it needs to get the components (n & e) of RSA key. If we store `Pkey` in Structure, every time we try to get n & e components, we will need transform it to RSA Key.